devices/Mpu6886/Mpu6886.cs

```diff
@@ -0,0 +1,419 @@
+        /// </summary>
+        public Vector3 GyroscopeOffset
+        {
+            get
+            {
+                SpanByte vec = new byte[6];
+                Read(Mpu6886.Register.GyroscopeOffsetAdjustmentXHighByte, vec);
+
+                Vector3 v = new Vector3();
+                v.X = (short)(vec[0] << 8 | vec[1]);
+                v.Y = (short)(vec[2] << 8 | vec[3]);
+                v.Z = (short)(vec[4] << 8 | vec[5]);
+
+                return v;
+            }
+
+            set
+            {
+                SpanByte registerAndOffset = new byte[7];
+                SpanByte offsetbyte = new byte[2];
+
+                registerAndOffset[0] = (byte)Mpu6886.Register.GyroscopeOffsetAdjustmentXHighByte;
+
+                BinaryPrimitives.WriteInt16BigEndian(offsetbyte, (short)value.X);
+                registerAndOffset[1] = offsetbyte[0];
+                registerAndOffset[2] = offsetbyte[1];
+
+                BinaryPrimitives.WriteInt16BigEndian(offsetbyte, (short)value.Y);
+                registerAndOffset[3] = offsetbyte[0];
+                registerAndOffset[4] = offsetbyte[1];
+
+                BinaryPrimitives.WriteInt16BigEndian(offsetbyte, (short)value.Z);
+                registerAndOffset[5] = offsetbyte[0];
+                registerAndOffset[6] = offsetbyte[1];
+
+                _i2c.Write(registerAndOffset);
+            }
+        }
+
+        /// <summary>
+        /// Reset the internal registers and restores the default settings. (Datasheet, page 47)
+        /// </summary>
+        public void Reset()
+        {
+            _i2c.Write(new SpanByte(new byte[] { (byte)Mpu6886.Register.PowerManagement1, 0b1000_0000 }));
+            Thread.Sleep(10);
+        }
+
+        /// <summary>
+        /// Set the chip to sleep mode. (Datasheet, page 47)
+        /// </summary>
+        public void Sleep()
+        {
+            _i2c.Write(new SpanByte(new byte[] { (byte)Mpu6886.Register.PowerManagement1, 0b0100_0000 }));
+            Thread.Sleep(10);
+        }
+
+        /// <summary>
+        /// Disables the sleep mode. (Datasheet, page 47)
+        /// </summary>
+        public void WakeUp()
+        {
+            _i2c.Write(new SpanByte(new byte[] { (byte)Mpu6886.Register.PowerManagement1, 0b0000_0000 }));
+            Thread.Sleep(10);
+        }
+
+        /// <summary>
+        /// Gets and sets the accelerometer full scale. (Datasheet page 37)
+        /// </summary>
+        public AccelerometerScale AccelerometerScale
+        {
+            get
+            {
+                SpanByte buffer = new byte[1];
+                Read(Mpu6886.Register.AccelerometerConfiguration1, buffer);
+                return (AccelerometerScale)buffer[0];
+            }
+
+            set
+            {
+                _i2c.Write(new SpanByte(new byte[] { (byte)Mpu6886.Register.AccelerometerConfiguration1, (byte)value }));
```

Comment:
if this is not the only element to setup in this register, you should first read the register, clear the bits related to what you want to setup and then apply the changes.
Similar as you did for AccelerometerLowPowerMode

Reply:
See previous comment, no problem now but avoid in the future I'll adjust.